Database variable bug

najimgh opened the pull request:
Issue ( #125 )
- Added declaration for openConnection variable

najimgh committed:
Issue (#125)
- Added declaration for openConnection variable

jeffbuonamici commented:
I'm getting a NullPointerException.

Cannot invoke "com.cbms.source.local.DatabaseConnection.getConnection()" because "com.cbms.source.local.Database.openConnection" is null

najimgh committed:
Issue (#125)
- Initialized openConnection variable in constructor of Database

najimgh committed:
Issue (#125)
- Database connection was being closed while the program was running.
- Added event listener on window close that will stop the connection to the database and close the program properly.

JayChouteau reviewed (approved):
The openConnection object gets declared now and the app does open properly. The change of implementation in the Main. java is also a better and more robust implementation and handling of our database connection relative to the app opening and closing. Good job.
